### ISSUE TITLE
Huang_2015: Processing submission

### ISSUE THREAD

**Issue by ehwenk** [traitecoevo/austraits.build#264]
**Source**: Huang et al (2015) Elevated temperature is more effective than elevated [CO2] in exposing genotypic variation in Telopea speciosissima growth plasticity: implications for woody
plant populations under climate change *Global Change Biology* http://doi.org/10.1111/gcb.12990
**Custodian**: David Tissue D.Tissue@westernsydney.edu.au and Renee Smith R.Smith@westernsydney.edu.au
**Handled by**: ?Austraits team member?

- [x] Contact made with contributor (label as `waiting on response` if waiting for files)
- [x] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [x] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [x] Metadata complete, builds into Austraits
- [x] Report reviewed by AusTraits team
- [x] Report sent to contributor ( label as `waiting on response` )
- [x] Response received from contributor
- [x] Files updated & report finalised
- [x] Final checks ( all tests pass )
- [ ] Branch merged into master branch (issue `closed` )

NB: 

- if you need to change the config files, e.g. to add a unit conversion or define a new trait. This should be done on the master branch (via pull or rebase)
